app/models/util/schedule.py

```diff
@@ -15,6 +15,10 @@ class WeekUser(NamedTuple):
     user_id: int
 
 
+def set_many(data: List[Tuple[int]]) -> List[Optional[Schedule]]:
```

Comment:
Unimplemented? Looks unused

Reply:
Ah... yes, it was an attempt to use SQLAlch batch edit features, but i couldn't find any CREATE OR UPDATE functionality that worked like i wanted, so.... yeah. I'll Delete

Reply:
Sg. I feel like it probably exists, but not sure what it would be